var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -21,11 +21,12 @@ class Singularity(MakefilePackage):
     '''
 
     homepage = "https://www.sylabs.io/singularity/"
-    url      = "https://github.com/sylabs/singularity/releases/download/v3.1.1/singularity-3.1.1.tar.gz"
+    url      = "https://github.com/sylabs/singularity/releases/download/v3.4.1/singularity-3.4.1.tar.gz"
```

Comment:
Base url does not change.  Spack automatically constructs the url based on the release version.

Reply:
ok will reverse this change

Reply:
Oh, good to know.  I thought there was a convention for keeping it at the first added version. 
 Sounds like the opposite is prefered, so I'll do that in the future.